torch/csrc/jit/script/sugared_value.h

```diff
@@ -172,12 +220,19 @@ struct MethodValue : public SugaredValue {
       at::ArrayRef<NamedValue> inputs,
       at::ArrayRef<NamedValue> attributes,
       size_t n_binders) override {
+    c10::optional<NamedValue> self;
+    if (auto userType = dynamic_cast<UserTypeValue*>(self_.get())) {
```

Comment:
Isn't it safe to just leave self as the first argument? The emit_call_to self logic is for silly builtin things where the self argument is not first.